Template/Components/Frame.html

```diff
@@ -10,22 +10,22 @@
         </header>
         <header id="sidebar">
             <nav>
-                <a href="[*Site Root*]index.html">Home</a>
-                <a href="[*Site Root*]About%20Us.html">About Us</a>
-                <a href="[*Site Root*]We%20Believe.html">We Believe</a>
-                <a href="[*Site Root*]Worship%20Services.html">Worship Services</a>
-                <a href="[*Site Root*]Sermons.html">Sermons</a>
-                <a href="[*Site Root*]Weekly%20Activities.html">Weekly Activities</a>
-                <a href="[*Site Root*]Events.html">Events</a>
-                <a href="[*Site Root*]Our%20Missions.html">Our Missions</a>
-                <a href="[*Site Root*]Contact%20Us.html">Contact Us</a>
+                <a siteReference="index.html">Home</a>
```

Comment:
`siteReference` is a pseudo‐attribute that represents a path relative to the site root. It unfolds into a real `href` attribute.